sprint/faq.md

```diff
@@ -0,0 +1,17 @@
+# Frequently Asked Questions
+
+## Do I need to be a PyMC user to participate in the open source working session?
+
+**Response:**  It is not necessary to be a PyMC user to contribute to this open source project. There are a number of issues where we could use your contribution. Examples:  
+- Adding timestamps to videos from our PyMCon conference
+- Updating docstrings in documentation
+- Update notebooks to v4.  This video, [Contributing to PyMC Documentation](https://youtu.be/fzpmLWQNj4A) provides a background and examples. 
```

Comment:
Updating notebooks to v4 requires being a PyMC user, and depending on the notebook an intermediate/advanced one even. The webinar and issue I suggested on hackmd are about updating the style and formatting of the notebooks. Adding metadata, hiding code cells (if they are only helper coding functions for example), adding alt text and captions to images... 

As it is usual for me, I'd also recommend linking to the webinar page using a sphinx cross reference, the video is embedded there directly too.